src/components/navbar.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import FuzzySearch from 'fuzzy-search';
+import { selectAllCovidData, searchData } from '../features/covid/covidSlice';
+
+function Navbar() {
+  const initialState = { search: '' };
+  const [search, setSearch] = useState(initialState);
+  const dispatch = useDispatch();
+  // const navigate = useNavigate();
+
+  const covidData = useSelector(selectAllCovidData);
+
+  const handleSubmitSearch = (data) => {
+    if (data) {
+      const searcher = new FuzzySearch(
+        covidData,
+        [
+          'item_id',
+          'deaths',
+          'country',
+          'continent',
+          'capital_city',
+          'confirmed',
+          'abbreviation',
+        ],
+        {
+          caseSensitive: false,
+        },
+      );
+      const result = searcher.search(data);
+      dispatch(searchData(result));
+      // navigate('/');
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.